doc/source/docs/user_guide/mapping.rst

```diff
@@ -222,6 +222,41 @@ We can set the ``zorder`` for cities higher than for world to move it of top.
     @savefig zorder_set.png
     world.plot(ax=ax, zorder=1);
 
+
+Pandas Plots
+-----------------
+
+Plotting methods also allows for different plot styles from pandas
+along with the default ``geo`` plot. These methods can be accessed using
+the ``kind`` keyword argument in :meth:`~GeoDataFrame.plot`, and include:
+
+* ``geo`` for mapping
+* ``line`` for line plots
+* ``bar`` or ``barh`` for bar plots
+* ``hist`` for histogram
+* ``box`` for boxplot
+* ``kde`` or ``density`` for density plots
+* ``area``  for area plots
+* ``scatter`` for scatter plots
+* ``hexbin`` for hexagonal bin plots
+* ``pie`` for pie plots
+
+.. ipython:: python
+
+    gdf = world.head(10)
+    @savefig pandas_line_plot.png
+    gdf.plot(kind='line')
+
+You can also create these other plots using the methods ``GeoDataFrame.plot.<kind>`` instead of providing the ``kind`` keyword argument.
```

Comment:
```suggestion
You can also create these other plots using the ``GeoDataFrame.plot.<kind>`` accessor methods instead of providing the ``kind`` keyword argument.
```